src/libraries/Microsoft.Extensions.Http/tests/DefaultHttpClientFactoryTest.cs

```diff
@@ -342,7 +342,7 @@ public async Task Factory_CreateClient_WithExpiry_HandlerCanBeReusedBeforeExpiry
             Assert.NotSame(activeEntry1.Handler, activeEntry2.Handler);
         }
 
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
+        [Fact]
```

Comment:
I just tried this locally and it still fails with a threading exception for me.

Reply:
You're probably not seeing it since the test suite crashes due to the two other `async Task` tests in this file that will need to be disabled.

Reply:
Readded the `ConditionalFact...`